src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -122,14 +123,25 @@ async def get_current_feeds(self, require_success: bool = False) -> Any:
                     )
                 )
         multi_call = Multicall(calls=calls, _w3=self.w3, require_success=require_success)
-        data = await multi_call.coroutine()
+        try:
+            data: Dict[str, Any] = await multi_call.coroutine()
+        except ValueError as e:
```

Comment:
You're expecting `ValueError`, but your logging messages say "KeyError"

Reply:
whoops!